ruby/.ruby-style.yml

```diff
@@ -908,14 +913,13 @@ Metrics/CyclomaticComplexity:
 
 Metrics/LineLength:
   Description: 'Limit lines to 100 characters.'
-  StyleGuide: 'Avoid lines longer than 100 characters'
   Max: 100
   Enabled: true
 
 Metrics/MethodLength:
   Description: 'Avoid methods longer than 10 lines of code.'
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#short-methods'
-  Enabled: true
+  Enabled: false
```

Comment:
Yes, I disabled it a while ago -- https://github.com/zestyzesty/backend/commit/477782d78b37a6168e0485b761884a80be7fc645

@matteddy brought up a good point that we should do an audit. Part of that should be centralizing the source, which I'm doing here.